crates/opencascade/src/primitives/wire.rs

```diff
@@ -43,6 +46,23 @@ impl Wire {
         Self::from_wire(make_wire.pin_mut().Wire())
     }
 
+    pub fn from_ordered_points(points: impl IntoIterator<Item = DVec3>) -> Result<Self, Error> {
+        let points: Vec<_> = points.into_iter().collect();
+        let mut make_wire = ffi::BRepBuilderAPI_MakeWire_ctor();
+
+        if points.len() < 3 {
```

Comment:
I think we can still accept 2 points - just create a segment and return it directly.

We can return `Error::NotEnoughPoints` for 1 or fewer points, though.

Reply:
Oh, I wrongly assumed that wires are closed sequence of edges. I now adjusted the implementation to allow using 2 points as well.